components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
```

Comment:
You'll want to use `@pipedream/platform` axios here. See https://pipedream.com/docs/components/guidelines/#use-pipedream-platform-axios-for-all-http-requests

Reply:
ok!